src/app/routes/routes.jsx

```diff
@@ -4,26 +4,23 @@ import React from 'react';
 import { IndexRoute, Route, Redirect } from 'react-router';
 
 // Import components
-import App from '../components/Application/Application';
-import Main from '../components/Application/Main';
-import BookPage from '../components/BookPage/BookPage';
-import Error404Page from '../components/Error404Page/Error404Page';
+import App from '../components/Application/Application.jsx';
```

Comment:
ESlint complains about having the `.jsx` or `.js` extensions. So let's remove them.